ssl/ssl_txt.c

```diff
@@ -34,10 +34,12 @@ int SSL_SESSION_print(BIO *bp, const SSL_SESSION *x)
     size_t i;
     const char *s;
     int istls13;
+    int isdtls13;
```

Comment:
There is no point in adding a variable if you actually need isversion13 variable.

Reply:
Agree!